fdb-record-layer-core/src/main/java/com/apple/foundationdb/record/provider/foundationdb/IndexingBase.java

```diff
@@ -253,51 +252,56 @@ private boolean shouldClearExistingIndexEntries(@Nonnull IndexState indexState,
     }
 
     @Nonnull
-    public IndexingBase setFallbackMode() {
-        fallbackMode = true;
-        return this;
+    public void setFallbackMode() {
```

Comment:
The method name should also be changed.

Reply:
To me, the logic seems to be that `OnlineIndexer` sets the `Indexer` fallback mode because it is called in the fallback path. The fact that the `Indexer` handles it by forcing a stamp overwrite is an implementation decision and shouldn't be the caller's concern. (Note that `OnlineIndexer` knows nothing about the indexing stamp)

If you disagree, though, I'll change it to `setForceStampOverwrite`.

Reply:
Maybe it's fine to call it `setFallbackMode`. But as a public method, you need a JavaDoc explain what it means and what the caller should expect the behavior to be when they set this option.